config/settings/production.yml

```diff
@@ -1,2 +1,6 @@
 # Use real authentication
 auth_provider: gds_sso
+
+mailchimp:
+  api_prefix: "us12"
+  lists: "c9cc8511c6,ef5e72c003"
```

Comment:
If we're defining the configuration here and not in the environment we can use a YAML array instead of having to split the string

Reply:
Good point!